data/strings/strings.txt

```diff
@@ -1596,16 +1596,16 @@
     ja = 利用を開始する前におおまかな世界地図をダウンロードします。これには%@の空き容量が必要です。
     ko = 시작하시기 전에 일반 세계 지도를 귀하의 장치로 다운로드하겠습니다.\n%@ 데이터가 필요합니다.
     lt = Leiskite mums atsisiųsti bendrąjį pasaulio žemėlapį į jūsų įrenginį prieš pradėdami naudotis programėle.\nTai užims %@ saugyklos.
-    lv = Pirms lietotnes izmantošanas lejupielādējiet kopējo pasaules karti.\nTā ierīce krātuvē aizņems %@ vietas.
+    lv = lv = Pirms sākt izmantot lietotni lejupielādējiet pasaules pārskata karti.\nTā ierīces krātuvē aizņems %@ vietas.
```

Comment:
This shouldn't have `lv = lv =`. Guys, when will we migrate to Weblate so that these mistakes don't occur? @pastk @biodranik @rtsisyk

[Carthago delenda est](https://en.wikipedia.org/wiki/Carthago_delenda_est)

Reply:
Well, as soon as someone will do all the migration work..